src/main/java/study/racingcar/domain/Car.java

```diff
@@ -0,0 +1,40 @@
+package study.racingcar.domain;
+
+public class Car {
+    private String name;
+    private Position position;
+
+    public Car(String name) {
+        this(name, 0);
```

Comment:
똑같은 기능의 객체를 매번 생성하기보다는 객체 하나를 재사용하는 것은 어떨까요?

```suggestion
        this(name, Position.ZERO);
```